internal_ws/yksg/src/lib.rs

```diff
@@ -191,6 +192,7 @@ struct StackFrame {
 pub struct StopgapInterpreter {
     /// Active stack frames (most recent last).
     frames: Vec<StackFrame>,
+    returnval: bool,
```

Comment:
I think we need to document what this means.

Reply:
s/returnval/return_val/ ?

Or just shorten it to `rv`?